src/SDKs/EventHub/EventHub.Tests/EventHub.Tests.csproj

```diff
@@ -1,29 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />  
+
   <PropertyGroup>
-    <PackageId>EventHub.Tests</PackageId>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>
     <Description>EventHub.Tests Class Library</Description>
     <Authors>Microsoft Corporation</Authors>
-    <AssemblyName>EventHub.Tests</AssemblyName>    
-  </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <AssemblyName>EventHub.Tests</AssemblyName>
+    <AssemblyOriginatorKeyFile>../keyfile.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageId>EventHub.Tests</PackageId>
```

Comment:
@v-Ajnava please let me know if you were having any difficulty in building the project, that prompted you for this change.
Please reach out to me and I can explain if things were either not building or understand what you wanted to do.